### ISSUE TITLE
[Feature] How to fetch custom outputs when converting to onnx and trt format?

### ISSUE THREAD

**Issue by goldentimecoolk** [open-mmlab/mmdeploy#2604]
### Motivation

Hi, I want to fetch more outputs from the pretrained pytorch model. For example, I want to add the features from backbone and the embeddings from encoder to the final outputs. Apart from the output_nams in `onnx_config = dict(output_names=['dets', 'labels'], input_shape=None)` and its related dynamic_axes if necessary, where should I make modications?

**Comment by yinfan98** [comment]
Hi, you mean custom ops? maybe you can see this folder
> https://github.com/open-mmlab/mmdeploy/tree/main/csrc/mmdeploy/backend_ops

**Comment by goldentimecoolk** [comment]
Hi @yinfan98, I think it is not the custom ops. I want to fetch more information other than `dets` and `labels` from the existing pretrained model.
